src/libraries/Microsoft.NET.WebAssembly.Threading/src/Microsoft.NET.WebAssembly.Threading.proj

```diff
@@ -0,0 +1,47 @@
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>none</DebugType>
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+    <IsPackable>true</IsPackable>
+    <!-- XXX FIXME: NU5128 suppresses a really scary message about missing net7.0 dependencies -->
+    <NoWarn>$(NoWarn);NU5128;NU5131</NoWarn>
+    <!-- Place assemblies in the nuget ref folder -->
+    <BuildOutputTargetFolder>ref</BuildOutputTargetFolder>
+    <!-- XXX FIXME: the rest of these props affect the NoTargets SDK in some way.  I want to be able to reference this project from another .csproj -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <GenerateDependencyFile>true</GenerateDependencyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Threading.Thread.WebAssembly.Threading\ref\System.Threading.Thread.WebAssembly.Threading.csproj" Pack="true" PrivateAssets="all" Private="true" IncludeReferenceAssemblyInPackage="true" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Threading.ThreadPool.WebAssembly.Threading\ref\System.Threading.ThreadPool.WebAssembly.Threading.csproj" Pack="true" PrivateAssets="all" Private="true" IncludeReferenceAssemblyInPackage="true" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.Tracing.WebAssembly.PerfTracing\ref\System.Diagnostics.Tracing.WebAssembly.PerfTracing.csproj" Pack="true" PrivateAssets="all" Private="true" IncludeReferenceAssemblyInPackage="true" />
+  </ItemGroup>
+
+  <!-- https://github.com/nuget/home/issues/3891 add the references .dlls to the package, and don't add the to the nuspec -->
+  <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="BuildOnlySettings;ResolveReferences">
+    <ItemGroup>
+      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths->WithMetadataValue('ReferenceSourceTarget', 'ProjectReference')->WithMetadataValue('PrivateAssets', 'All'))" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_ExperimentalUpdateFileVersion"
+          AfterTargets="_InitializeAssemblyVersion">
+    <PropertyGroup>
+      <_FileVersionMaj>$(FileVersion.Split('.')[0])</_FileVersionMaj>
+      <_FileVersionMin>$(FileVersion.Split('.')[1])</_FileVersionMin>
+      <_FileVersionBld>$(FileVersion.Split('.')[2])</_FileVersionBld>
+      <_FileVersionRev>$(FileVersion.Split('.')[3])</_FileVersionRev>
+      <FileVersion>$(_FileVersionMaj).$([MSBuild]::Add($(_FileVersionMin), 100)).$(_FileVersionBld).$(_FileVersionRev)</FileVersion>
```

Comment:
I assume you copied and pasted this from somewhere else but I hate seeing it without a comment that explains why we even do it